packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 ## v3.1.0-dev (Aug 08, 2024)
 
 -   Remove ocapi session-bridging on phased launches [#2011](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2011)
-
 -   Add `defaultDnt` to support setting the dnt flag for SLAS. Upgrade `commerce-sdk-isomorphic` to v3.1.1 [#1979](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1979)
 -   Update logout helper to work for guest users [#1997](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1997)
+- Update `useCustomMutation` hook to accept body as mutate parameter [#2008](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2008)
```

Comment:
I think we need to update the [feature-passwordless-social-login](https://github.com/SalesforceCommerceCloud/pwa-kit/tree/feature-passwordless-social-login) branch and merge the latest updates to this branch